controller_joint_names: ['', 'right_front_omni', 'right_back_omni', 'left_back_omni', 'left_front_omni', 'joint1', 'joint2', ]

controller_manager:
  ros:
    auto_start: true
    update_rate: 100
  
joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 50
arm_1_controller:
  type: position_controllers/JointPositionController
  joint: joint1

arm_2_controller:
  type: position_controllers/JointPositionController
  joint: joint2



gazebo_ros_control:
  pid_gains:
    right_front_omni: {p: 1.0, i: 0.0, d: 0.0}
    left_front_omni: {p: 1.0, i: 0.0, d: 0.0}
    right_back_omni: {p: 1.0, i: 0.0, d: 0.0}
    left_back_omni: {p: 1.0, i: 0.0, d: 0.0}
    joint1: {p: 1, i: 0.01, d: 0.0}
    joint2: {p: 1, i: 0.01, d: 0.0}